### DISCUSSION TITLE
is there a way to put the rpn stack on top and change the index direction from bottom to top?

### DISCUSSION THREAD

**Discussion by c33s** [Qalculate/qalculate-gtk#489] [Q&A] [answered]
![image](https://github.com/Qalculate/qalculate-gtk/assets/649209/cec6beec-b436-4930-acc6-bf27e793e45f)

**Comment by hanna-kn** [comment, marked as answer]
No.

**Reply by c33s** [reply]
@hanna-kn does it make sense to open an issue or is this feature to hard to implement?
